nft-market/constants/en.global.json

```diff
@@ -0,0 +1,28 @@
+{
+  "navigation": {
+    "nftMarketplace": "NFT Marketplace",
+    "nearAccountText": "NEAR Account:",
+    "mint": "Mint",
+    "myNFTs": "My NFTs",
+    "buyNFTs": "Buy NFTs",
+    "login": "Login",
+    "logout": "Logout"
+  },
+  "home": {
+    "title": "NFT Marketplace"
+  },
+  "mint": {
```

Comment:
Add some more descriptions for buttons and functions -> like on mint page add title and descrtion with something like -> "You are about to mint an NFT token with default nft metadata ....". 
Also look into minting NFT with just using "new" method and passing your own metadata"

Reply:
done that